src/test/run-make/incr-prev-body-beyond-eof/Makefile

```diff
@@ -1,5 +1,5 @@
 -include ../../run-make-fulldeps/tools.mk
-
+# ignore-32bit
```

Comment:
Can we annotate this with a FIXME or link to an issue?

Reply:
Yes, I was thinking the same. I'm not seeing an existing issue, but I can create one. @jonas-schievink, please let me know if you know of an existing issue for this.

Reply:
No, I didn't open one